recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas_%.bbappend

```diff
@@ -67,6 +67,23 @@ KERNEL_DEVICETREE_salvator-x-h3-xt = " \
     renesas/r8a7795-salvator-x-4x2g-doma.dtb \
 "
 
+##############################################################################
+# Salvator-X M3
+###############################################################################
+# N.B. DomA device tree is reused from Salvator-X H3 ES3.0 4x2G
+###############################################################################
+SRC_URI_append_salvator-x-m3-xt = " \
+    file://r8a7796-salvator-x-dom0.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
+    file://r8a7796-salvator-x-domd.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
+    file://r8a7795-salvator-x-4x2g-doma.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
```

Comment:
Why `4x2G`? Why not just `8G`?

Reply:
As per my understanding this is for historical reasons: Renesas has named with "4x2g" DT files for boards with 8GiB memory

Reply:
I have added comment